projects/ngx-intl-tel-input/src/lib/ngx-intl-tel-input.component.ts

```diff
@@ -26,7 +28,7 @@ import { phoneNumberValidator } from './ngx-intl-tel-input.validator';
 	// tslint:disable-next-line: component-selector
 	selector: 'ngx-intl-tel-input',
 	templateUrl: './ngx-intl-tel-input.component.html',
-	styleUrls: ['./ngx-intl-tel-input.component.css'],
+	styleUrls: ['./bootstrap-dropdown.css', './bootstrap-tooltip.css', './ngx-intl-tel-input.component.css'],
```

Comment:
```bootstrap-tooltip.css``` is not accessed, because of encapsulation by ```bs-tooltip-container``` component.
This results in an unstyled tooltip. One solution is to use ```encapsulation: ViewEncapsulation.None```, but that would leak all styles globally, which is not nice. Another is to clone the whole tooltip into our project...

Reply:
Ahh, drats, missed the tooltip testing! Sorry about that!

I'd personally vote to moving the tooltip into the project, as it's only ~400 LOC to add. What do you think?

Reply:
Yes, I think that way we could keep it like we want. And maybe even improve the design in the future. If you could do that I would appreciate it a lot!

Reply:
I plan on making those changes in the next day or two, but in the meantime, do we also want to bring in the `ngx-bootstrap` dropdown as well? It is also ~300 LOC across its codebase and it might be nice to eliminate the `ngx-bootstrap` dependency entirely

Reply:
Yes, that has been a long standing issue :)

Reply:
Hello @pasevin ,

Do you have any update concerning this improvement? It's something we would need for our project. Is there a way we could help on getting this done? 

Thanks,

Reply:
Not sure about the progress. @crutchcorn began working on it, and I don't want to pressure him :) Hope he will complete it at some point. I personally don't want to interfere with his PR.

Reply:
hi, any updates on this issue?